docs/src/main/tut/kleisli.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: default
+title:  "Kleisli"
+section: "data"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/data/Kleisli.scala"
+scaladoc: "#cats.data.Kleisli"
+---
+# Kleisli
+Kleisli is a data type that will come in handy often, especially if you are working with monadic functions.
```

Comment:
Hm I think this is covered by the next section under "Functions" - or did you have something different in mind?